repos/system_upgrade/el7toel8/actors/postgresqlcheck/libraries/library.py

```diff
@@ -0,0 +1,81 @@
+from leapp.models import InstalledRedHatSignedRPM
+from leapp.libraries.common import reporting
+from leapp.libraries.stdlib import CalledProcessError, api, run
+
+def get_postgresql_packages():
+    """ Get list of installed PostgreSQL packages """
+
+    for rpm_pkgs in api.consume(InstalledRedHatSignedRPM):
+        return {pkg.name for pkg in rpm_pkgs.items if pkg.name.startswith('postgresql')}
+
+def get_postgresql_extensions():
+    """ Get list of PostgreSQL extensions """
+
+    result = []
+    try:
+        output = run([
+            '/usr/bin/su',
+            '-l', 'postgres'
+            '-c', 'psql -tAc \'\\dx\''
+        ], split=True)['stdout']
+    except CalledProcessError:
+        api.current_logger().warning('Unable to list PostgreSQL extensions: psql failed to run')
+        return None
+
+    for entry in output:
+        entry = entry.strip()
+        if not entry:
+            continue
+        name, version, schema, description = entry.split('|')
+        result.append({'name': name,
+                       'version': version,
+                       'schema': schema,
+                       'description': description})
+
+    return result
+
+def generate_report(packages):
+    """ Generate reports informing user about possible manual intervention requirements """
+
+    DOC_URL = 'https://access.redhat.com/articles/4055641'
+    MISSING_EXTENSIONS = ['dummy_seclabel', 'test_parser', 'tsearch2']
+
+    extensions = get_postgresql_extensions()
+
+    # Show documentation url if postgresql-server installed
+    if 'postgresql-server' in packages:
+        title = 'PostgreSQL server installation detected'
+        severity = 'high'
+        summary = 'PostgreSQL server component will be upgraded. Since RHEL-8 includes PostgreSQL\n' \
+                  'server 10 by default, which is incompatible with 9.2 included in RHEL-7, it is\n' \
+                  'necessary to proceed with additional steps for complete upgrade of the\n' \
+                  'PostgreSQL data. Back-up your data before proceeding with the upgrade and read\n' \
+                  'more about upgrade steps at {}.'.format(DOC_URL)
+
+        reporting.report_generic(title=title, severity=severity, summary=summary)
```

Comment:
Please use `report_with_links` if you want to include links - https://github.com/oamg/leapp-repository/blob/master/repos/system_upgrade/el7toel8/libraries/reporting.py#L40

Reply:
I've changed the report to `report_with_links`.